### PR TITLE
Traces v -09 (C_R authenticated in message_2)

### PR DESCRIPTION
**Pull request by marco-tiloca-sics**
This PR updates the EDHOC traces according to https://github.com/lake-wg/edhoc/pull/455

### CODE REVIEW

**Review by gselander — APPROVED** [review]
I haven't checked any values, but the things that were changes and the sizes looks right. Thanks for adding clarification in headings too at the end.

File: draft-ietf-lake-traces.md [hunk 1]
```diff
@@ -2618,23 +2618,23 @@ where hash_length is the length in bytes of the output of the EDHOC hash algorit
...
+07 58 20 c9 02 b1 e3 a4 32 6c 93 c5 55 1f 5f 3a a6 c5 ec c0 24 68 06
+76 56 12 e5 2b 5d 99 e6 05 9d 6b 6e 18 20
 ~~~~~~~~
 
 ~~~~~~~~
 PRK_out (Raw Value) (32 bytes)
-6b 2d ae 40 32 30 65 71 cf bc 2e 4f 94 a2 55 fb 9f 1f 3f b2 9c a6 f3
-79 fe c9 89 d4 fa 90 dc f0
+2c 71 af c1 a9 33 8a 94 0b b3 52 9c a7 34 b8 86 f3 0d 1a ba 0b 4d c5
+1b ee ae ab df ea 9e cb f8
```
Comment (malishav) [inline draft-ietf-lake-traces.md:2637]: OK

File: draft-ietf-lake-traces.md [hunk 2]
```diff
@@ -2666,8 +2666,8 @@ info for PRK_exporter (CBOR Sequence) (4 bytes)
 
 ~~~~~~~~
 PRK_exporter (Raw Value) (32 bytes)
-4f 0a 5a 82 3d 06 d0 00 5e 1b ec da 8a 6e 61 f3 c8 c6 7a 8b 15 da 7d
-44 d3 58 5e c5 85 4e 91 e2
+e1 4d 06 69 9c ee 24 8c 5a 04 bf 92 27 bb cd 4c e3 94 de 7d cb 56 db
+43 55 54 74 17 1e 64 46 db
```
Comment (malishav) [inline draft-ietf-lake-traces.md:2670]: OK

File: draft-ietf-lake-traces.md [hunk 3]
```diff
@@ -2735,7 +2735,7 @@ info for OSCORE Master Secret (CBOR Sequence) (3 bytes)
 
 ~~~~~~~~
 OSCORE Master Secret (Raw Value) (16 bytes)
-8c 40 9a 33 22 23 ad 90 0e 44 f3 43 4d 2d 2c e3
+f9 86 8f 6a 3a ca 78 a0 5d 14 85 b3 50 30 b1 62
```
Comment (malishav) [inline draft-ietf-lake-traces.md:2738]: OK

File: draft-ietf-lake-traces.md [hunk 4]
```diff
@@ -2764,7 +2764,7 @@ info for OSCORE Master Salt (CBOR Sequence) (3 bytes)
 
 ~~~~~~~~
 OSCORE Master Salt (Raw Value) (8 bytes)
-61 63 f4 4b e8 62 ad fa
+ad a2 4c 7d bf c8 5e eb
```
Comment (malishav) [inline draft-ietf-lake-traces.md:2767]: OK

**Review by malishav — APPROVED** [review]
Hi,

I went through the second trace manually and I couldn't spot any error in what I could verify with our implementation. Thanks for providing the PR!

**Comment by gselander** [comment]
Marek confirmed too. Merging this for -09.

**Pull request merged**